packages/main/src/RPA/Salesforce.py

```diff
@@ -176,6 +217,12 @@ def _require_authentication(self) -> None:
         if self.sf is None:
             raise SalesforceAuthenticationError("Authentication is not completed")
 
+    def _require_no_session(self) -> None:
```

Comment:
```suggestion
    def _raise_on_session(self) -> None:
```
if you like it more.